Source/RxSwift/and.swift

```diff
@@ -76,7 +76,7 @@ extension ObservableType where Element == Bool {
 						completed += 1
 						guard completed == collection.count else { return }
 						// if all emitted at least one `true`, emit true otherwise just complete
-						if emitted.reduce(true, { $0 && $1 }) {
+						if emitted.allSatisfy({ $0 }) {
```

Comment:
```suggestion
						if emitted.allSatisfy { $0 } {
```

Reply:
Thank you for your comments :)
But I am not sure that this code will compile successfully

Reply:
Hey - I believe it will, why wouldn't it? It's a regular trailing closure

![image](https://user-images.githubusercontent.com/605076/60179190-c8b49300-9825-11e9-8fb5-8b4d879434f9.png)

Reply:
Ah apparently it doesn't work in the context of `if`s. TIL :)